src/components/DateMonthPicker/index.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react'
+import { Button, translate, Icon, Bold } from 'cozy-ui/transpiled/react'
+import range from 'lodash/range'
+import { format } from 'date-fns'
+
+const gridStyle = {
+  display: 'grid',
+  gridTemplateColumns: 'repeat(4, auto)',
+  gridTemplateRows: 'repeat(3, 3fr)',
```

Comment:
If all rows get the same amount of available space, `1fr` is not sufficient? (not blocking, just for my culture)